src/DisputeResolution/index.js

```diff
@@ -21,6 +21,7 @@ class DisputeResolution extends Component {
     if (this.props.isFetching) return false
 
     const dispute = this.props.caseData
+    console.log(dispute)
```

Comment:
remove this log

Reply:
Can you fix the build on netlify and travis?

Reply:
fixed